integration-tests/salesforce/src/test/java/org/apache/camel/quarkus/component/salesforce/SalesforceIntegrationTest.java

```diff
@@ -33,16 +35,17 @@
 public class SalesforceIntegrationTest {
 
     @Test
-    public void testChangeDataCaptureEvents() {
+    public void testCDCAndStreamingEvents() {
```

Comment:
Is there any requirements/simplifications to test CDC and streaming in the same place ?

Reply:
@aldettinger  There is no requirement. 
Just that all the consumers CDC, Streaming without RAW payload, and streaming with payload, will get the same created Account.
If we want to test those separately, I guess I need to order them or run in different classes.
Is this answering your question ?

Reply:
Yes, I think I get it. The key point is that the Salesforce events mechanism is kind of shared by design. So I don't think there is a need to split, it looks clear now.